pkgs/development/python-modules/jaxlib/bin.nix

```diff
@@ -211,12 +205,14 @@ buildPythonPackage {
     scipy
   ];
 
-  # Note that cudatoolkit is snecessary since jaxlib looks for "ptxas" in $PATH.
-  # See https://github.com/NixOS/nixpkgs/pull/164176#discussion_r828801621 for
-  # more info.
+  # jaxlib looks for ptxas at runtime, eg when running `jax.random.PRNGKey(0)`.
+  # Linking into $out is the least bad solution. See
+  # * https://github.com/NixOS/nixpkgs/pull/164176#discussion_r828801621
+  # * https://github.com/NixOS/nixpkgs/pull/288829#discussion_r1493852211
+  # for more info.
   postInstall = lib.optional cudaSupport ''
-    mkdir -p $out/bin
-    ln -s ${cudatoolkit}/bin/ptxas $out/bin/ptxas
+    mkdir -p $out/${python.sitePackages}/jaxlib/cuda/bin
+    ln -s ${lib.getBin cudaPackagesGoogle.cuda_nvcc}/bin/ptxas $out/${python.sitePackages}/jaxlib/cuda/bin/ptxas
```

Comment:
I keep posting cosmetic suggestions and that feels wrong, but `lib.getExe' cudaPackagesGoogle.cuda_nvcc "ptxas"` 😆

Reply:
no, it's great! i'm learning new `lib` functions this way!

done in https://github.com/NixOS/nixpkgs/pull/288829/commits/9e8cdafeb8387a8f2bc3179a12d93f184b2347cd